recipes/configure-logstash-kibana.rb

```diff
@@ -1,38 +0,0 @@
-# Cookbook Name:: mh-opsworks-recipes
```

Comment:
this functionality was split up and added to the respecting logstash/kibana install recipes